FND/Places/Addresses.rdf

```diff
@@ -99,29 +120,457 @@
+						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
+					</owl:Restriction>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddressLine2"/>
+						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
+					</owl:Restriction>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddressLine3"/>
+						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
+					</owl:Restriction>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+						<owl:onClass rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
+						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+					</owl:Restriction>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:subClassOf>
+		<rdfs:label>conventional street address</rdfs:label>
+		<skos:definition>physical address that identifies a specific building and unit or location within a building on a street to which communications may be delivered</skos:definition>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-adr;Floor">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitDesignator"/>
+		<rdfs:label>floor</rdfs:label>
+		<skos:definition>all of the rooms or areas on the same level of a building; a story</skos:definition>
+		<fibo-fnd-plc-adr:requiresSecondaryUnitRange rdf:datatype="&xsd;boolean">true</fibo-fnd-plc-adr:requiresSecondaryUnitRange>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;GeographicDirectionalSymbol">
+		<rdfs:subClassOf rdf:resource="&lcc-lr;CodeElement"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;hasTag"/>
+				<owl:someValuesFrom rdf:resource="&xsd;string"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>geographic directional symbol</rdfs:label>
+		<skos:definition>code element that gives directional information for postal delivery</skos:definition>
+		<skos:example>In the United States, these include N, S, E, W, NE, NW, SE, SW.</skos:example>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;InternationalAddress">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-plc-cty;hasCountry"/>
+				<owl:onClass rdf:resource="&lcc-cr;Country"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>international address</rdfs:label>
+		<skos:definition>physical address located outside the country of origin</skos:definition>
```

Comment:
Removed and added the restriction to physical address as optional.

Reply:
That's improved but should it not be a equivalentClass restriction? So that any address with a hasCountry property is automatically an InternationalAddress. Personally I think this whole class has no value: people could always query for the presence of the property. Would one also define "AddressIncludingState", "AddressIncludingStreet"?

Reply:
I'll remove the class.  It is important to the US standard as a separate class, and I'll move it there.